### DISCUSSION TITLE
questions on time-to-first binding and lifetime distribution graphs

### DISCUSSION THREAD

**Discussion by gelles-brandeis** [gelles-brandeis/tapqir#377] [Q&A] [answered]
1) I'm assuming the graphs are cumulative distributions (e.g., survival curves).  Maybe put "cumulative: in axis labels to make this clear?

2) Is there a way to recover the individual lifetimes that are being fit (so the user can fit to an arbitrary different model).

**Comment by ordabayevy** [comment, marked as answer]
Hi @gelles-brandeis 

1. This has been fixed in #410 
2. Individual time-to-first binding lifetimes are saved as a table (with rows as posterior sample numbers and columns as AOI numbers) in the `{model name}_ttfb-data-points-channel{number}.csv` file

**Comment by gelles-brandeis** [comment]
Does the program also save the posterior-sampled dwell times?

**Comment by ordabayevy** [comment]
Yes, it is saved in the `{model name}_dwelltime-intervals-channel{number}.csv` file
